tests/client-dbus/tests/dbus/_dm.py

```diff
@@ -62,9 +57,9 @@ def remove_stratis_setup():
     # There is some dependency ordering with regards to dm tables, we will
     # simply iterate over the list attempting to remove them until they are
     # all gone or we give up trying.
-    while devices and attempts <= max_loops:
+    while devices != [] and attempts <= max_loops:
         attempts += 1
-        for dev in list(devices):
+        for dev in devices:
```

Comment:
@mulkieran The reason I did a `list(devices)` was to ensure we are not traversing a list that we are modifying, eg.:

```python
a = [ str(x) * 10 for x in range(0, 10)] 
for z in a:
    a.remove(z)

print("Ending Length ", len(a))

a = [ str(x) * 10 for x in range(0, 10)] 
for z in list(a):
    a.remove(z)

print("Ending Length ", len(a))

```
Yields
```
Ending Length  5
Ending Length  0
```